docs/guides/manage-identities/05_identity-schema.mdx

```diff
@@ -0,0 +1,293 @@
+---
+id: identity-schema
+title: Identity schema
+---
+
+The Identity schema is a JSON schema that describes the data model that makes up the Identity. In Ory, every Identity can have its
+own model (a unique set of data fields).
+
+The Identity schema can either be one of the available presets or a [customized model](#custom-identity-model).
+
+## Available presets
+
+Ory Cloud provides three basic Identity Schema presets.
+
+### Username & password
+
+This preset is useful for applications that don't need the user's email address and don't prioritize a high degree of user
+anonymity.
+
+:::warning
+
+This preset disables account verification, account recovery, and account enumeration defenses. Don't use it in security-sensitive
+environments.
+
+:::
+
+With this preset, every Identity has a single trait - the `username`. The `username` is the login identifier:
+
+```json5
+// Identity example
+{
+  id: "6e9d3d30-f93e-4630-901f-c2096953723d",
+  traits: {
+    username: "some-username",
+  },
+}
+```
+
+### Email & password
```

Comment:
```suggestion
### Email and password
```

should we ban ampersand, I think that would make it easier to handle, wdyt? we have a wild mix atm :D